tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,8 @@
-const App = () => {
-	return <div>A tic-tac-toe game</div>;
-};
+const App = () => (
+  <main>
+    <h1>Noughts and Crosses</h1>
+    <GameBoard />
```

Comment:
Each component has it's own module. 👍 Keeping the `<GameBoard/>` and `<GameCell` separate from each other and the `<App/>` allows for the code to more maintainable.